Description: Put all orphan files into the airport2config package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/airport2config/airport/Airport2InfoElement.java
===================================================================
--- airport-utils-2.orig/airport2config/airport/Airport2InfoElement.java	2010-03-11 20:55:56.300753470 +0100
+++ airport-utils-2/airport2config/airport/Airport2InfoElement.java	2010-03-11 20:56:29.493753675 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airport;
 
 import java.io.*;
 
Index: airport-utils-2/airport2config/airport/Airport2InfoElementVector.java
===================================================================
--- airport-utils-2.orig/airport2config/airport/Airport2InfoElementVector.java	2010-03-11 20:56:29.271753361 +0100
+++ airport-utils-2/airport2config/airport/Airport2InfoElementVector.java	2010-03-11 20:56:29.510965972 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airport;
 
 import java.util.*;
 import java.io.*;
Index: airport-utils-2/airport2config/AboutDialog.java
===================================================================
--- airport-utils-2.orig/airport2config/AboutDialog.java	2010-03-11 20:55:56.319753144 +0100
+++ airport-utils-2/airport2config/AboutDialog.java	2010-03-11 20:56:29.536150668 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airport2config;
 
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/airport2config/AirportBaseStation128BitConfigurator.java
===================================================================
--- airport-utils-2.orig/airport2config/AirportBaseStation128BitConfigurator.java	2010-03-11 20:55:56.328753661 +0100
+++ airport-utils-2/airport2config/AirportBaseStation128BitConfigurator.java	2010-03-11 20:56:29.552904383 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airport2config;
 
 
 import java.util.*;
Index: airport-utils-2/airport2config/AirportBaseStationCLI.java
===================================================================
--- airport-utils-2.orig/airport2config/AirportBaseStationCLI.java	2010-03-11 20:55:56.337753258 +0100
+++ airport-utils-2/airport2config/AirportBaseStationCLI.java	2010-03-11 20:56:29.577995934 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airport2config;
 
 
 import java.util.*;
Index: airport-utils-2/airport2config/AirportBaseStationConfigurator.java
===================================================================
--- airport-utils-2.orig/airport2config/AirportBaseStationConfigurator.java	2010-03-11 20:55:56.346753529 +0100
+++ airport-utils-2/airport2config/AirportBaseStationConfigurator.java	2010-03-11 20:56:29.594760695 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airport2config;
 
 
 import java.util.*;
Index: airport-utils-2/airport2config/AirportBaseStationConfiguratorMicro.java
===================================================================
--- airport-utils-2.orig/airport2config/AirportBaseStationConfiguratorMicro.java	2010-03-11 20:55:56.355753495 +0100
+++ airport-utils-2/airport2config/AirportBaseStationConfiguratorMicro.java	2010-03-11 20:56:29.636752293 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airport2config;
 
 
 import java.util.*;
Index: airport-utils-2/airport2config/AirportBaseStationMultiConfigurator.java
===================================================================
--- airport-utils-2.orig/airport2config/AirportBaseStationMultiConfigurator.java	2010-03-11 20:55:56.367759580 +0100
+++ airport-utils-2/airport2config/AirportBaseStationMultiConfigurator.java	2010-03-11 20:56:29.653246058 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airport2config;
 
 
 import java.util.*;
